test/app-tap/module_api.c

```diff
@@ -3072,16 +3158,18 @@ luaopen_module_api(lua_State *L)
 		{"test_box_ibuf", test_box_ibuf},
 		{"tuple_validate_def", test_tuple_validate_default},
 		{"tuple_validate_fmt", test_tuple_validate_formatted},
-		{"test_key_def_dup", test_key_def_dup},
-		{"tuple_field_by_path", tuple_field_by_path},
-		{"test_decimal", test_decimal},
-		{"decimal_mul", test_decimal_mul},
-		{"decimal_div", test_decimal_div},
-		{"isdecimal", test_isdecimal},
-		{"isdecimal_ptr", test_isdecimal_ptr},
+		{"test_key_def_dup",           test_key_def_dup},
+		{"tuple_field_by_path",        tuple_field_by_path},
+		{"test_decimal",               test_decimal},
+		{"decimal_mul",                test_decimal_mul},
+		{"decimal_div",                test_decimal_div},
+		{"isdecimal",                  test_isdecimal},
+		{"isdecimal_ptr",              test_isdecimal_ptr},
```

Comment:
Please avoid unrelated changes. There's no need to reformat this table.

Reply:
Fixed, sorry, forgot to overlook the diff.